chart/kubeapps/README.md

```diff
@@ -242,6 +242,38 @@ kubectl apply -f <repo name>.yaml
 
 After that you should be able to access the new version of Kubeapps. If the above doesn't work for you or you run into any other issues please open an [issue](https://github.com/kubeapps/kubeapps/issues/new).
 
+### Upgrading to 5.0.0
```

Comment:
Can you rename this title? (In the other title it says "Upgrading to 2.0" (talking about the app version, not the chart cersion). We can use this PR to also release the new version so this can be: `Upgrading to 2.0.1 (Chart 5.0.0)`.